.github/workflows/test_build_deploy.yml

```diff
@@ -95,7 +95,8 @@ jobs:
             {
               "edge": { "dev_env": "wire-webapp-edge" },
               "dev": { "dev_env": "wire-webapp-dev" },
-              "master": { "dev_env": "wire-webapp-master" }
+              "master": { "dev_env": "wire-webapp-master" },
+              "tmp_qa_migration": { "qa_env": "wire-webapp-qa-al2-migration" }
```

Comment:
```suggestion
```
Let's not auto deploy to QA env in the build script (the `deploy-to-test-env.yml` is made for this)